content/300-guides/200-deployment/110-deployment-guides/500-deploying-to-netlify.mdx

```diff
@@ -6,68 +6,87 @@ metaDescription: 'Learn how to deploy Node.js and TypeScript applications that a
-
 </TopBlock>
 
 ## Prerequisites
 
-- GitHub account
-- Hosted PostgreSQL database and a URL from which it can be accessed, e.g. `postgresql://username:password@your_postgres_db.cloud.com/db_identifier` (you can use Heroku, which offers a [free plan](https://dev.to/prisma/how-to-setup-a-free-postgresql-database-on-heroku-1dc1)).
-- [Netlify](https://app.netlify.com/signup) account connected to your GitHub account (Netlify will need access to the repository you will create as part of this guide).
-- [Netlify CLI](https://docs.netlify.com/cli/get-started/) installed.
-- Node.js installed.
+- a GitHub account
+- a hosted PostgreSQL database and a connection string for it, such as `postgresql://username:password@your_postgres_db.cloud.com/db_identifier` (you can use Supabase, which offers a [free plan](https://dev.to/prisma/set-up-a-free-postgresql-database-on-supabase-to-use-with-prisma-3pk6)).
+- a [Netlify](https://app.netlify.com/signup) account connected to your GitHub account (Netlify needs access to the repository you will create as part of this guide).
+- [Netlify CLI](https://docs.netlify.com/cli/get-started/) installed
+- Node.js installed
 
 ## Prisma workflow
 
-Prisma supports different workflows depending on whether you integrate with an existing database or create a new one from scratch. Regardless of the workflow, Prisma relies on the Prisma schema - that is, the `schema.prisma` file.
+Prisma supports different workflows depending on whether you integrate with an existing database or create a new one from scratch. Regardless of the workflow, Prisma relies on the Prisma schema, that is, the `schema.prisma` file.
+
+With an empty database in place, you continue with the Prisma workflow that involves pushing the data model defined in the `schema.prisma` file to the database and then generating Prisma Client.
 
-This guide starts with an empty database that you must have already created and looks as follows:
+The steps are outlined below.
 
-1. Migrate the existing Prisma schema to your existing database using `prisma db push`.
+1. Push the existing Prisma schema to your existing database using `prisma db push`.
 2. Run `prisma generate` to generate the Prisma Client based on the Prisma schema.
 
 ## 1. Fork the <inlinecode>deployment-example-netlify</inlinecode> repository
 
-Go to [`deployment-example-netlify`](https://github.com/prisma/deployment-example-netlify) and fork the repository by clicking the **Fork** button in the top-right corner:
+To begin, fork the repository of the deployment example project [`deployment-example-netlify`](https://github.com/prisma/deployment-example-netlify).
+
+With a forked repository, you create a copy of the repository to which you will be able to commit your own changes. Later, you will connect the forked repository to Netlify so that changes are automatically built and deployed.
```

Comment:
```suggestion
With a forked repository, you create a copy of the repository to which you can commit your own changes. Later, you connect the forked repository to Netlify so that changes are automatically built and deployed.
```